.github/workflows/wheels.yml

```
@@ -0,0 +1,32 @@
+jobs:
+  build-wheels:
+    name: Build wheels for ${{ matrix.python }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ["3.10", "3.11", "3.12"]
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python ${{ matrix.PYTHON_VERSION }}
```

actionlint

property "python_version" is not defined in object type {python: number}

(expression)